### DISCUSSION TITLE
终于登上绳网啦，我要成为传奇绳匠

### DISCUSSION THREAD

**Discussion by Legend648** [share121/inter-knot#2666] [其他]
![1723734664718](https://github.com/user-attachments/assets/2bbab438-5e58-4844-a8b3-5163e2f9237b)
分享美图

**Comment by Philia093-T** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/12/28/d6bfa59a83721d7e7dd27363f5bfc88b_732370910818099450.png" alt="芙宁娜-好戏！" width="75" >好看！

**Comment by qiu-p** [comment]
好看！

**Comment by AnkaAlioth** [comment]
好看<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >
